scripts/tools/eslint-plugin-prettier-internal-rules/test.js

```diff
@@ -308,6 +308,7 @@ test("no-identifier-n", {
   ],
 });
 
+/* **
```

Comment:
minor nit: I think it would be ideal if we could find a way to get these tests working with the updates I made to fix `npm clean dev test`.